### ISSUE TITLE
OneOS 2.0.0 Public Beta Buggy scrollbar in Sketch

### ISSUE THREAD

**Issue by Andy73** [oeed/OneOS#318]
![oneos05](https://cloud.githubusercontent.com/assets/7235381/6426825/cf0f1e3c-bf6c-11e4-84ac-d7c24d728161.png)
As you can see, the scrollbar touches the bottom of the screen while the content didn't reach the bottom yet (Invert filter is not visible). Try it for yourself, dragging the scrollbar with mouse causes this behaviour.

**Comment by oeed** [comment]
Yeah that's a Bedrock issue. I've been meaning to redo the scroll mechanics for a while.
